Makefile

```diff
@@ -12,7 +12,7 @@ export GOPROXY
 
 PKGS=$(shell go list ./... | grep -v -E '/vendor/|/test|/examples')
 GOLANG_FILES:=$(shell find . -name \*.go -print)
-ASSETS=$(shell grep -oh 'assets/.*\.yaml' pkg/manifests/manifests.go)
+ASSETS=$(shell grep -oh '[^"]*/.*\.yaml' pkg/manifests/manifests.go | sed -s 's/^/assets\//')
```

Comment:
@paulfantom it seems that the Makefile doesn't detect properly when to regenerate asset files now. I'm trying this but open to better ideas :)

Reply:
Let's see if this work on non linux env, sed tends to be different from the gnu sed on Linux. Will test and report back.

Reply:
Yes as I suspected, this does not work on mac:
```
 ➜  cluster-monitoring-operator git:(86d11142) ✗ make generate
sed: illegal option -- s
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
sed: illegal option -- s
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
/Users/lili/go/src/github.com/openshift/cluster-monitoring-operator/tmp/bin/embedmd -w `find Documentation -name "*.md"
````

I am fine with merging it now as it fixes a bug, but we should follow up with a fix to make it work in all env.

Reply:
Ho I don't need the `-s` option (here you can hint that I've copied the first answer from Stackoverflow...). I've updated the change and it should be compatible with all seds hopefully.